llama-index-integrations/storage/chat_store/llama-index-storage-chat-store-azure/llama_index/storage/chat_store/azure/base.py

```diff
@@ -408,13 +416,15 @@ def _get_default_metadata(self, key: str) -> dict:
             "MessageCount": 0,
         }
 
-    def _get_or_default_metadata(self, metadata_client: TableClient, key: str) -> dict:
+    async def _get_or_default_metadata(
+        self, metadata_client: TableClient, key: str
```

Comment:
`TableClient` import in line 7 is unsuitable here.
`from azure.data.tables.aio import TableClient` is better.